pandas/core/frame.py

```diff
@@ -812,7 +812,7 @@ def to_string(
     # ----------------------------------------------------------------------
 
     @property
-    def style(self):
+    def style(self) -> "Styler":
```

Comment:
mypy complains about this, needs Styler imported in the TYPE_CHECKING block

Reply:
sorry not sure what you mean, can you please explain a bit

Reply:
nvm, fixed figured it out.